doc-botonic/docs/faq.md

```diff
@@ -0,0 +1,36 @@
+---
+id: faq
+title: FAQ
+---
+
+
+><details><summary><b>How does Botonic and Hubtype relate to each other?</b></summary>
+Botonic is an open source project created by the Hubtype team. Hubtype is a startup that helps companies engage with their customers on messaging apps.
+Hubtype offers the best developer experience when it comes to deploying and integrating your Botonic bot.
+</details>
+
+><details><summary><b>How do I integrate a chat in a website?</b></summary>
+Once you deploy a bot,  you get a link to integrate different channels, including the "webchat" channel, which is a JavaScript snippet that you can embed in your website.
+</details>
+
+><details><summary><b>Can I edit my chat  styles?</b></summary>
+This action is not possible except for webviews as some of these apps (like Facebook Messenger) allow you to embed html pages inside their chat window.
+Hubtype provides support for web chat (JS SDK) and in-app chat (iOS/Android SDK). In these cases, you can customize the look and feel of the chat window.
+</details>
+
+><details><summary><b>How do I delete a bot?</b></summary>
+Go to https://app.hubtype.com/bots/all and select your bot. Then click on the **Settings** tab and on **Delete this bot**.
+</details>
+
+><details><summary><b>How to deal with NLP/NLU?</b></summary>
+You must integrate Dialogflow or Watson. The integration allows you to use regular expressions to route intents to actions. Check out the NLU example in our github repo.
+</details>
```

Comment:
This has changed. Now we should encourage the use of our NLU plugin and then mention that you can integrate Dialogflow/Watson/Luis/Inbenta too.

Reply:
FAQ updated.